Dockerfile

```diff
@@ -55,11 +55,11 @@ RUN directory-artifact-gen \
     -t raspberrypi3 \
     -t raspberrypi4 \
     -t raspberrypi \
-    -d /var/www/localhost \
+    -d /data/www/localhost \
     -o mender-demo-artifact.mender \
-    /var/www/localhost \
+    /data/www/localhost \
     -- \
-    --software-filesystem rootfs \
+    --software-filesystem data-partition \
```

Comment:
Should this be `data` instead?

Reply:
Do we have an explicit policy/guideline about the name of the data partition filesystem?
I was unsure about it, too. I like data, but here we mention `data-partition`:
https://docs.mender.io/artifact-creation/software-versioning/advanced-versioning#advanced-versioning

Reply:
to me it doesn't matter if it's `data` or `data-partition` as long as we're consistent, which, in this case, means we should use `data-partition` here or change the docs

Reply:
Yes. This was more like an open question to make us all aware (I did pre-approve). I think is okey to merge as-is.